src/qtgui/plotter.cpp

```diff
@@ -1375,20 +1375,45 @@ void CPlotter::drawOverlay()
     {
         QList<BandInfo> bands = BandPlan::Get().getBandsInRange(m_CenterFreq + m_FftCenter - m_Span / 2,
                                                                 m_CenterFreq + m_FftCenter + m_Span / 2);
-
+        int bandLevel = 0;
+        qint64 last_max = 0;
+        qint64 last_max_0 = 0;
+        
+        std::sort(bands.begin(), bands.end(), []( const BandInfo& band1 , const BandInfo& band2 )->bool {
```

Comment:
There is already `bool operator<(const BandInfo &other) const` operator, consider extending it and use here instead of the lambda.

Reply:
Done.